Kickstarter-iOS/Views/CommentCellHeaderStackView.swift

```diff
@@ -71,8 +71,8 @@ internal final class CommentCellHeaderStackView: UIStackView {
 
   // MARK: - Configuration
 
-  internal func configureWith(comment: Comment, user: User?) {
-    self.viewModel.inputs.configureWith(comment: comment, user: user)
+  internal func configureWith(comment: Comment, user: User?, project: Project) {
```

Comment:
Does `CommentCellHeaderStackView`need the project?

Reply:
Good point. I don't think any of them other than `CommentCell` needs it. Going to remove that parameter.

Reply:
You either make `Project` optional or you create separate input for the other `Cell` and  `CommentCellHeaderStackViewneed `.

Reply:
Yep, updated!